packages/mlbdd/mlbdd.0.1/opam

```diff
@@ -0,0 +1,11 @@
+opam-version: "1.2"
+maintainer: "Arlen Cox <arlencox@gmail.com>"
+authors: "Arlen Cox <arlencox@gmail.com>"
+homepage: "https://github.com/arlencox/mlbdd"
+bug-reports: "https://github.com/arlencox/mlbdd/issues"
+license: "MIT"
+dev-repo: "git@github.com:arlencox/mlbdd.git"
```

Comment:
This needs to be `https://githubcom/arlencox/mlbdd.git` or else only you can access it.